python/discovery/README.md

```diff
@@ -0,0 +1,74 @@
+How does endpoint discovery work?
+===
+The mechanism to discover the Scheduled Events Service IP address is to resend a DHCP request. In the centralized DHCP service, the response is customized 
+by carrying the IP address inside the DHCP response option 245. The endpoint discovery script uses the same mechanism to determine 
+the Scheduled Events IP address. Once the IP address is obtained, it is made available as part of the environment variable both in Windows and Linux worlds.
+
+Running the discovery script
+===
+### `Pre-requesites`: Install python 2.4+ 
+
+### `Windows:`
+
+Once you have installed python, open an admin cmd window. Run the script by calling “python discovery.py”. 
+If you want to include any of the optional switches, you can do so by calling “python discovery.py –debug”.
+
+### `Linux:`
+
+Once you have installed python, run the script by calling “sudo python discovery.py”. You can include any/all of the 
+optional switches.
+
+<br>
+
+Discovering the endpoint
+===
+By default, upon discovering the Scheduled Events Service IP address, the script adds it to the environment variables. The variable name in both Windows and 
+Linux is “CLOUDCONTROLIP”. However, the script has two additional modes of operation – “--debug” and “--donotaddtoenv” as listed below.
```

Comment:
Should the CLOUDCONTROLIP be changed to SCHEDULEDEVENTSIP?

Reply:
Yep! Will review this in your separate PR :)